### ISSUE TITLE
Cannot parse if line ends with linebreak and space

### ISSUE THREAD

**Issue by zemanlx** [hadolint/language-docker#35]
```Dockerfile
FROM centos:centos7

RUN yum -y install --setopt=tsflags=nodocs epel-release && \ 
    yum -y update && yum clean all
```

There is a space ` ` after `\`

```bash
$ integration-tests/parseFile Dockerfile.break
Left (TrivialError (SourcePos {sourceName = "Dockerfile.break", sourceLine = Pos 4, sourceColumn = Pos 5} :| []) (Just (Tokens ('y' :| ""))) (fromList [Tokens ('#' :| ""),Label ('A' :| "DD"),Label ('A' :| "RG"),Label ('C' :| "MD"),Label ('C' :| "OPY"),Label ('E' :| "NTRYPOINT"),Label ('E' :| "NV"),Label ('E' :| "XPOSE"),Label ('F' :| "ROM"),Label ('H' :| "EALTHCHECK"),Label ('L' :| "ABEL"),Label ('M' :| "AINTAINER"),Label ('O' :| "NBUILD"),Label ('R' :| "UN"),Label ('S' :| "HELL"),Label ('S' :| "TOPSIGNAL"),Label ('U' :| "SER"),Label ('V' :| "OLUME"),Label ('W' :| "ORKDIR"),Label ('t' :| "he rest of a new line followed by the next instruction"),EndOfInput]))
```

It wasn't catched in integration tests because error message has changed 
https://github.com/hadolint/language-docker/blob/9a0e80fe0dfec001d2c02310d0c1798c10289297/integration-tests/parse_files.sh#L112

**Comment by zemanlx** [comment]
@lorenzo we shouldn't release hadolint until this one is fixed

**Comment by lorenzo** [comment]
Thanks for noticing, this should be easy to fix. I’ll release a new version of the parser today
